app/controllers/api/v3/matches_controller.rb

```diff
@@ -99,7 +99,7 @@ def multi_create
       render json: {status: 400, message: e.message} and return
     end
 
-    Delayed::Job.enqueue MatchJobs::CreateNewMatchesJob.new(@req[:matches].map(&:symbolize_keys), deck, current_user.id)
+    Delayed::Job.enqueue MatchJobs::CreateNewMatchesJob.new(@req[:matches].map(&:symbolize_keys), deck.id, deck.klass_id, current_user.id)
```

Comment:
Can we just put `Match.delay.mass_import_new_matches(matches_params, deck_id, klass_id, user_id)` here instead? Since you got it down into one method in the model.